src/Nuget/Sarif.Sdk.nuspec

```diff
@@ -4,11 +4,11 @@
     <id>$id$</id>
     <version>$major$.$minor$.$patch$$prerelease$</version>
     <title>.NET Static Analysis Results Interchange Format (SARIF) Sdk</title>
-    <authors>Billy O'Neal,Larry Golding,Michael C. Fanning,Nick Guerrera</authors>
+    <authors>Billy O'Neal, Larry Golding, Michael C. Fanning, Nick Guerrera</authors>
```

Comment:
Changed <authors> to "Microsoft Corporation"; left owners as is except I put your name first.

---
In reply to: [65741879](https://github.com/Microsoft/sarif-sdk/pull/320#discussion_r65741879) [](ancestors = 65741879)